### DISCUSSION TITLE
Listbox: <Item> with non-plain text contents is unsupported by type to select for accessibility. Please add a `textValue` prop.

### DISCUSSION THREAD

**Discussion by J4v4Scr1pt** [heroui-inc/heroui#2394] [Help] [answered]
I really need some help with this. I'm using NextUI Pro and the SideBar component.

But when I list my menu items I get a lot of errors as show in the title.
Even the example on the Pro homepage has this error.
How can I reslove this? 

![image](https://github.com/nextui-org/nextui/assets/5929055/ce0ace30-f988-4484-9ce0-b787b5597cc3)

**Comment by kikiou1991** [comment, marked as answer]
make sure you are passing an aria-label prop to the components that you are using, it usually hapopens with dropdwon, dropwdwonsection etc..

**Comment by yanfer** [comment]
I had exactly the same issue, and the answer is very simple DO NOT ERASE  `aria-label` IN THE `Table` component!!!

`<Table
      aria-label="Example table with dynamic content" 
      etc...`

 // you can change the string but not the prop, with this I hadn't the same issue again
